memory_core/genesis/memory_core.vp

```diff
@@ -197,7 +197,7 @@ always @(*) begin
       mem_ren1 = lb_ren;
```

Comment:
Let's keep .vp changes out of functional python changes. If we need to clean up .vp files, let's do in a separate pass.